return {
    BinarySearch = require("./BinarySearch-Features");
    Cut = require("./Cut-Features");
    Equals = require("./Equals-Features");
    Filter = require("./Filter-Features");
    FoldLeft = require("./FoldLeft-Features");
    FoldRight = require("./FoldRight-Features");
    GroupBy = require("./GroupBy-Features");
    Insert = require("./Insert-Features");
    IsOrdered = require("./IsOrdered-Features");
    Map = require("./Map-Features");
    Mean = require("./Mean-Features");
    Merge = require("./Merge-Features");
    MergeMany = require("./MergeMany-Features");
    MutableBinaryInsert = require("./MutableBinaryInsert-Features");
    MutableMerge = require("./MutableMerge-Features");
    MutableMergeMany = require("./MutableMergeMany-Features");
    MutableReverse = require("./MutableReverse-Features");
    MutableShuffle = require("./MutableShuffle-Features");
    Percentile = require("./Percentile-Features");
    Product = require("./Product-Features");
    Range = require("./Range-Features");
    Remove = require("./Remove-Features");
    RemoveRange = require("./RemoveRange-Features");
    Reverse = require("./Reverse-Features");
    SelectFirst = require("./SelectFirst-Features");
    SelectLast = require("./SelectLast-Features");
    SelectRandom = require("./SelectRandom-Features");
    SelectWeighted = require("./SelectWeighted-Features");
    Shuffle = require("./Shuffle-Features");
    Sort = require("./Sort-Features");
    Sum = require("./Sum-Features");
}